elfi/bo/acquisition.py

```diff
@@ -42,15 +42,15 @@ def evaluate(self, x, t=None):
         """
         return NotImplementedError
 
-    def acquire(self, n_values, pending_locations=None, t=None):
+    def acquire(self, n_values, pending_locations=None, t=0):
```

Comment:
Nothing too much, just to be safe. Since it is supposed to be a iteration id number coming from an algorithm, setting it to a fixed valid value can cause hidden bugs. You cannot know if it was truly set by the caller or was just 0 by accident.